test/jdk/javax/swing/AbstractButton/5049549/bug5049549.java

```diff
@@ -39,18 +35,40 @@
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 
+/* @test
+   @bug 5049549 7132413
+   @summary Tests that the proper icon is used for different states.
+   @library ../../regtesthelpers
+   @build Blocker
+   @run main/manual bug5049549
+*/
+
 public class bug5049549 {
```

Comment:
Since you're moving it, I suggest formatting it according to conventions used in most tests:
```suggestion
/*
 * @test
 * @bug 5049549 7132413
 * @summary Tests that the proper icon is used for different states.
 * @library ../../regtesthelpers
 * @build Blocker
 * @run main/manual bug5049549
 */
public class bug5049549 {
```
That is with starting asterisk on each line, and align the last asterisk.